test/helpers/integration-server.ts

```diff
@@ -13,7 +13,6 @@ export const startServer = async (command: string): Promise<PID> => {
     })
     const serverPid = serverProcess.pid as PID
     serverProcess.stdout.on("data", (data) => {
-      console.log(data.toString())
```

Comment:
this is helpful (necessary imo) otherwise the stdout of the subprocess are just eaten

Reply:
fixed